spec/service_worker/index.bs

```diff
@@ -1781,17 +1789,44 @@ spec: webidl; urlPrefix: https://heycam.github.io/webidl/
                     </li>
                     <li>Else:
                       <ol>
-                        <li>Let <var>potentialResponse</var> be a copy of <var>response</var>'s associated <a href="https://fetch.spec.whatwg.org/#concept-response-response">response</a>, except for its body.</li>
+                        <li>Let <var>bytes</var> be an empty byte sequence.
+                        <li>Let <var>end-of-body</var> be false.
```

Comment:
It's set when all data is pulled from _reader_. We need to run these subsubsteps until all data is pushed to _newStream_.